README.md

```diff
@@ -27,3 +27,95 @@ graph LR;
     pred--P(B)-->team2_win;
     pred--P(C)-->draw;
 ```
+
+## Problem Formulation
+
+This is essentially a multi-class classification problem, where the number of classes $K=3$.
+
+The expected output of the model is the predicted probability for each class:
+* $P(A) \in [0, 1]$: The probability for event $A$ that *Team1* wins the game.
+* $P(B) \in [0, 1]$: The probability for event $B$ that *Team2* wins the game.
+* $P(C) \in [0, 1]$: The probability for event $C$ that the game ends in a draw.
+
+Note that $P(A) + P(B) + P(C) = 1$.
+
+## The Model
+
+The model is a standard Transformer which follows the general encoder-decoder framework.
```

Comment:
position embedding's input and struct

Reply:
I basically followed the original paper, where the PE part does not have any explicit input.